### ISSUE TITLE
include Malcolm's letter front and centre front page

### ISSUE THREAD

**Issue by kat** [AOGPN/opengovernment.org.au#105]
(via Peter Timmins) it's important that people know this initiative is advocated at the highest level of government and may tip the balance for those wishing to take part

**Comment by kat** [comment]
this is a 'nice to have'

**Comment by equivalentideas** [comment]
Here's a few ideas for different ways to do this:

## 1. Link in the site introduction text

Include a reference and link to this letter, or a blog post about the letter, in the site introduction text ( see #36 ). This means that people who are just learning about the ogp and network should read and discover it. Unfortunately we haven't fixed #44 so links are still quite hard to see, which kind undermines this solution for me. We could fix that bug if we think this is the simplest/best solution.

![screen shot 2016-02-09 at 2 35 43 pm](https://cloud.githubusercontent.com/assets/1239550/12907016/78ce6d6e-cf3a-11e5-8e49-a5970629fcd4.png)

## 2. Include it in the updates sections via blog post

Do a blog post about it so it appears in the _latest posts_ section near the top of the page and in the larger _Latest updates_ section, with a attention grabbing featured image, further down the front page. We could make this post “Sticky” so it stays in the first position if we want.

![screen shot 2016-02-09 at 2 18 21 pm](https://cloud.githubusercontent.com/assets/1239550/12906816/0ee77c08-cf38-11e5-93a0-9132f4814438.png)

## 3. Add a front page section for the letter

Add a front page section before or after the _Working groups_ section on the front page that is all about the letter. We could give it more content/context if we did this. We could also add an image (something better than in this sketch :point_down: )

![screen shot 2016-02-09 at 2 02 44 pm](https://cloud.githubusercontent.com/assets/1239550/12906847/637ef340-cf38-11e5-975d-84809a5eca1d.png)
![screen shot 2016-02-09 at 2 01 47 pm](https://cloud.githubusercontent.com/assets/1239550/12906848/63818d8a-cf38-11e5-9534-f118215349b9.png)

## 4. Replace the “latest posts” section with an introduction to this letter

![screen shot 2016-02-09 at 2 29 47 pm](https://cloud.githubusercontent.com/assets/1239550/12906947/a5d63b44-cf39-11e5-9c11-4e7341db0e91.png)

Next to the site introduction text we could replace the _Latest posts_ section with a little box about this. If we did this it would be instead of #37 for now at least.

What ever we do we should do a blog post about the letter included an image of it like [Peter’s post](http://foi-privacy.blogspot.com.au/2015/12/pm-ogp-goals-directly-align-with.html). We should include machine readable text of the letter for accessibility/search-ability.

Do you think any of these are on the right track @kat ? There’s a range of what we might call “front and centre” here.

**Comment by kat** [comment]
some strong contenders there
Do either of you find one more compelling over the others?

**Comment by kat** [comment]
also I'll ask Peter Timmins for feedback

**Comment by henare** [comment]
Here's my thoughts:
1. A bit subtle - as I understand it, Peter wanted it more prominent on our site
2. Similarly subtle. It's just a post rather than its own, important thing
3. Another big section makes the page noisier
4. I like this the best. It's prominent enough and replaces a section we weren't sure about anyway

**Comment by equivalentideas** [comment]
> Do either of you find one more compelling over the others?

@kat I think the first option is in a 'just in time' kinda place where you get it in the context of what the network is. I also think the the third option could work well with some good text: It's kinda like “This thing has the tick of approval and backing of the Prime Minister”.

> are there any that are faster to do than others, or any that would be thornier to implement

Not particularly.

**Comment by kat** [comment]
I like the second version of 3, and I like the look of 4 but something about having two "Read More" boxes in parallel seems distracting. Also I doubt we're allowed to use the Gov coat of arms. https://www.dpmc.gov.au/pmc/publication/commonwealth-coat-arms-information-and-guidelines

**Comment by equivalentideas** [comment]
For me the key question is whether the letter is more important that the working groups?

I just thought of another thing @kat When do we want people to discover the letter? When they just discover the Network? When the become members? When they're learning about the OGP? When they join the working groups? I think we're coming at this from a page layout perspective rather than from the users perspective.

**Comment by kat** [comment]
It's something Peter thinks is important when referring someone to learn about the network, that they understand that it has this high level (and recent) backing.

**Comment by kat** [comment]
(via Peter) Three or four- use as heading "Prime Minister Turnbull recommits Australia to the Open Government Partnership" I'm with Luke " It's kinda like “This thing has the tick of approval and backing of the Prime Minister”. I've been thinking the themes or this might be better on the landing page than the Working Groups.

**Comment by equivalentideas** [comment]
> It's something Peter thinks is important when referring someone to learn about the network, that they understand that it has this high level (and recent) backing.

This makes me think it's also important to make this really prominent on the About page, by mentioning it in the main introduction to the network.

It also suggests that solution number 1 might be a good idea since people who are just learning about the network are going to read that prominent text and possibly head straight to the about page. They might not even look around at other sections of the home page. I've fixed the link style bug #44 that was making the link invisible, so it could now appear to users like this:

![screen shot 2016-02-09 at 4 48 13 pm](https://cloud.githubusercontent.com/assets/1239550/12908573/0e9736b6-cf4d-11e5-92e2-4563a08e4c83.png)

What do you think @kat ? I'm a bit hesitant to add a new section when we've previously said that the main function of the home page is to get people into contributing to a commitment through the working groups as quickly as possible. Also adding a nice new section to the home page feels good, but would it actually be better for users to simplify the page and put this important information in some good succinct text about the core of the network?

**Comment by equivalentideas** [comment]
Ok, after discussing this with @kat, the prime purpose of this is for people who get sent to the site to find out about the network to see something that says “This thing has the tick of approval and backing of the Prime Minister”.

I'm going to go [with the solution 4 above](https://github.com/AOGPN/opengovernment.org.au/issues/105#issuecomment-181692079), because it keeps working groups high in the hierarchy of content on the home page, but also displays the letter as one of the most important things people need to know. This means that once people learn about the letter they can still see that the working groups are also important.

**Comment by equivalentideas** [comment]
I've implemented this now:

![screen shot 2016-02-10 at 4 59 04 pm](https://cloud.githubusercontent.com/assets/1239550/12939837/f05f0752-d017-11e5-948b-f384ce6b66cb.png)

It links to a blog post with the letter https://opengovernment.org.au/2015/11/28/the-prime-ministers-letter-reconfirms-australias-commitment-to-ogp/

This is all hard coded into the theme at the moment. If this is something we want to play with more I could link it to the admin interface so it can be edited there, but that felt like too much work for now.

With the text I wanted to make it formal enough to reenforce that this is official by mentioning the dates and also link it back to the action plan, which the letter talks about, and remind people of the timeline.

> PM Turnbull renews OGP commitment
> On 24 November 2015, Prime Minister Malcolm Turnbull wrote to the Open Government Partnership. He renewed Australia’s commitment to becoming an OGP member and launched work towards our first National Action Plan due July 2016.

The image is of the MP’s signature on the letter—_It’s official yall_.
